NEWS.md

```diff
@@ -144,6 +151,11 @@ menu.border.color: #aaaaaa
 
 ### Changed
 
+- Temporarily set xwaylandPersistence to 'yes' by default to avoid a bug that is
+  present in wlroots <0.18.2 and which has the potential to crash the compositor
+  when performing a drag-and-drop action at the same time as the XWayland server
+  is shutting down. This will be reverted when wlroots-0.18.2 can be linked
+  with. [#2371] [#2414]
```

Comment:
Could be rewritten to something like

> Set xwaylandPersistence default value to `yes` when compiled with wlroots < 0.18.2. This prevents a bug which has the  potential to crash the compositor when performing a drag-and-drop action at the same time as the XWayland server is shutting down.

The revert part isn't required anymore as with 0.18.2 and later we will be back to `no` by default automatically.

Reply:
Thanks. Done.